### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/Ring/Basic.lean [hunk 1]
```diff
@@ -60,37 +60,40 @@ end AddHomClass
...
+  toFun := (r * ·)
   map_zero' := mul_zero r
   map_add' := mul_add r
 #align add_monoid_hom.mul_left AddMonoidHom.mulLeft
 
 @[simp]
-theorem coe_mul_left [NonUnitalNonAssocSemiring R] (r : R) :
-    (mulLeft r : R → R) = HMul.hMul r :=
+theorem coe_mulLeft [NonUnitalNonAssocSemiring R] (r : R) :
+    (mulLeft r : R → R) = (r * ·) :=
```
Comment (ChrisHughes24) [inline Mathlib/Algebra/Ring/Basic.lean:71]: I'd be more inclined not to change this one. Somebody might want to write `rw [<- coe_mulLeft]` and that becomes harder with the new statement.

**Comment by ChrisHughes24** [comment]
Apart from the one comment LGTM.

bors d+

File: Mathlib/Algebra/Ring/Basic.lean (hunk 1 above)
Comment (Ruben-VandeVelde) [reply Mathlib/Algebra/Ring/Basic.lean:71]: Yeah, that's what I added `mulLeft_apply` for. In fact, your PR #2725 introduced this change relative to mathlib3. I think it's better to be consistent with mathlib3 as well as with `coe_mulRight`.

**Comment by semorrison** [comment]
@Ruben-VandeVelde, could you explain why you undelegated and asked for review again?

**Comment by Ruben-VandeVelde** [comment]
bors merge

**Comment by Ruben-VandeVelde** [comment]
bors merge

**Pull request closed without merging**